src/themes/greyVest/index.js

```diff
@@ -451,6 +451,7 @@ export let GVStyle = () => (
       }
       .contexture-facet > label {
         margin: 5px 0;
+        max-width: 340px;
```

Comment:
does this look ok in QueryBuilder?

Reply:
not really

<img width="1008" alt="Screen Shot 2019-08-07 at 11 59 21 AM" src="https://user-images.githubusercontent.com/35466670/62638156-d96b2500-b90a-11e9-92ba-0f5dfb79941f.png">

we should be able to keep the truncation on variable width with some [CSS tweaks](https://stackoverflow.com/questions/12649904/css-text-ellipsis-when-using-variable-width-divs), though